clients/js/README.md

```diff
@@ -1,97 +1,95 @@
 # Wormhole CLI
 
-
+This tool is a command line interface to Wormhole.
 ## Installation
 
     make install
 
 This installs two binaries, `worm-fetch-governance` and `worm` on your `$PATH`.
 
-## Usage
-
-### `worm-fetch-governance`
-
-    Usage:
-      worm-fetch-governance [sequence]
-
-    Fetch a governance VAA by sequence number, and print it as hex.
-
-
-For example
-
-    worm-fetch-governance 13940208096455381020
-
-prints
-
-    01000000010d0012e6b39c6da90c5dfd3c228edbb78c7...
-
-
-### `worm`
-
-This is the main CLI tool. To use it, set up `$HOME/.wormhole/.env` with your
+To use `worm`, set up `$HOME/.wormhole/.env` with your
 private keys, based on `.env.sample` in this folder.
 
-    worm [command]
+## Usage
+
+``` sh
+worm [command]
 
-    Commands:
-      worm generate      generate VAAs (devnet and testnet only)
-      worm parse <vaa>   Parse a VAA
-      worm submit <vaa>  Execute a VAA
+Commands:
+  worm generate                             generate VAAs (devnet and testnet
+                                            only)
+  worm parse <vaa>                          Parse a VAA
```

Comment:
mention the format the vaa's can be in (hex and b64, right?) and if anything is needed to specify the format as a flag when passing in

Reply:
yeah, both hex and base64 are accepted, and no additional flags are needed to specify -- updated the help string